src/librustc/mir/interpret/value.rs

```diff
@@ -346,6 +353,12 @@ impl<'tcx, Tag> Scalar<Tag> {
         Ok(b as u64)
     }
 
+    pub fn to_u128(self) -> EvalResult<'static, u128> {
+        let sz = Size::from_bits(128);
+        let b = self.to_bits(sz)?;
+        Ok(b)
```

Comment:
The intermediate variable `b` seems pointless.

Reply:
Ah, it is. It's because I was copy/paste/editting similar code from rustc.